mono/metadata/object.c

```diff
@@ -6241,6 +6248,13 @@ mono_array_new_specific_checked (MonoVTable *vtable, uintptr_t n, MonoError *err
 	return (MonoArray*)o;
 }
 
+
+MonoArrayHandle
+mono_array_new_specific_handle (MonoVTable *vtable, uintptr_t n, MonoError *error)
+{ // FIXMEcoop
```

Comment:
What needs to be fixed here?

Also nit: I don't think comments on the same line as a brace is Mono coding style.

Reply:
1. Use of underlying raw pointer, maybe? Though see previous point -- Centrinel should guide is eventually, once the obvious pieces are reduced.
2. Maybe it could/should be? Plus it is a fixme so ideally gets deleted.

Reply:
so ok I'll just remove it